libmcount/plthook.c

```diff
@@ -764,7 +764,7 @@ unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
 
 	if (likely(child_idx < pd->dsymtab.nr_sym)) {
 		sym = &pd->dsymtab.sym[child_idx];
-		pr_dbg3("[idx: %4d] enter %lx: %s (mod: %lx)\n",
+		pr_dbg3("[idx: %4d] enter %llx: %s@plt (mod: %lx)\n",
```

Comment:
The preferred way to do that is using `%PRIx64` (although it looks ugly).

Reply:
Sure. I will use it.

Reply:
I changed this part to `PRIx64`. Thanks!